preppal-be/routes/recipeApi.ts

```diff
@@ -165,7 +165,7 @@ routerRecipeApi.post("/createRecipe", async (req, res) => {
         const user = await Author.findOne({ username: author });
         const ownRecipes = user.ownRecipes;
         ownRecipes.push(recipe.id);
-        const updatedUser = await Author.findByIdAndUpdate(user.id, { ownRecipes });
+        // const updatedUser = await Author.findByIdAndUpdate(user.id, { ownRecipes });
```

Comment:
If this was causing the linter error:

`const updatedUser = await Author.findByIdAndUpdate(user.id, { ownRecipes });`
to
`await Author.findByIdAndUpdate(user.id, { ownRecipes });`

I think we still need this line to keep the user's list updated

Reply:
yeah that's why i left the method call. But if you're not using the variable, you don't need to declare it

Reply:
the issue was my importing of FilterValues